clipboard/ios/Plugin/Clipboard.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import Capacitor
+
+public class Clipboard: NSObject {
```

Comment:
If this is no longer exposed to Obj-C, it does not need to inherit from NSObject. Doing so means it can be instantiated from Obj-C but none of its methods can be called.
```suggestion
public class Clipboard {
```